lib/ProgressPlugin.js

```diff
@@ -537,9 +542,8 @@ class ProgressPlugin {
 				handler(0, "");
 			}
 		});
```

Comment:
You probably want to change that `compiler.hooks.initialize` to `compiler.hooks.beforeRun`, instead of filtering 0% in the default logger with `wasLogged`